account_fiscal_year_closing/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# Copyright 2016-2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Fiscal year closing",
+    "summary": "Generic fiscal year closing wizard",
+    "version": "11.0.1.0.0",
+    "category": "Accounting & Finance",
+    "website": "https://www.tecnativa.org/",
```

Comment:
website -> https://github.com/OCA/account-closing/

Reply:
Thank you. I would ask @pedrobaeza what does he think, not a specialist in these matters myself.

Reply:
Yes, it's OK to change to GitHub OCA websites :)